README.md

```diff
@@ -3854,20 +3854,38 @@ resource cleanup when possible.
 <sup>[[link](#percent-s)]</sup>
 
 * <a name="percent-literal-braces"></a>
-  Prefer `()` as delimiters for all `%` literals, except `%r`. Since parentheses
-  often appear inside regular expressions in many scenarios a less common
-  character like `{` might be a better choice for a delimiter, depending on the
-  regexp's content.
-<sup>[[link](#percent-literal-braces)]</sup>
+  Prefer different kinds of braces for percent literals:
+  <sup>[[link](#percent-literal-braces)]</sup>
+  - `()` for string literals(`%q`, `%Q`).
+  - `[]` for array literals(`%w`, `%i`, `%W`, `%I`) as it is aligned with the standard array literals.
+  - `{}` for regexp literals(`%r`) since parentheses often appear inside regular expressions. In many scenarios a less common
```

Comment:
Re-flow this long line to 80 chars.

In many scenarios -> That's why a less common character with `{` is usually the best delimiter for `%r` literals.